hack/terraform/aws/variables.tf

```diff
@@ -87,6 +95,39 @@ variable "aws_vm_os_types_agented" {
   ]
 }
 
+variable "aws_vm_win_os_types_agented" {
+  type = list(object({
+    name            = string
+    login           = string
+    init            = string
+    ami_name_search = string
+    ami_owner       = string
+  }))
+  default = [
+    {
+      name            = "windows2019-1"
+      login           = "Administrator"
+      init            = "init_script_windows.ps1"
+      ami_name_search = "NepheWindows2019Image4"
```

Comment:
Create a new image and remove Image4 from it.. Push this to all region and create a doc mentioning how to create a image like this

Reply:
Will do it in next version and update the PR

Reply:
Done. Updated the PR.